blocks/form/form.css

```diff
@@ -313,7 +317,7 @@ main .connectus .form-consent {
 }
 
 /* stylelint-disable-next-line  selector-class-pattern */
-main .connectus :is(.form-company, .form-Pardot_Form_Message__c, .form-description, .form-consent) {
+main .connectus :is(.form-companyName, .form-Pardot_Form_Message__c, .form-description, .form-consent) {
```

Comment:
Marketo expects the field name to be company. Since company H1 is already exists in same page accessibility score was less and @solaris007 changed it to companyName.

At time of generating payload we should replace companyName to company.

@vdua - We decided to use the name of field as component id, should we add prefix (may in other project) to avoid such conflicts?

Reply:
Hi @jalagari,
Sorry, I am not able to understand what exactly needs to be done.